mail_show_follower/models/res_config_settings.py

```diff
@@ -9,3 +9,50 @@ class ResConfigSettings(models.TransientModel):
         related="company_id.show_internal_users_cc",
         readonly=False,
     )
+    show_followers_message_sent_to = fields.Html(
+        string="Text 'Sent to'",
+        related="company_id.show_followers_message_sent_to",
+        translate=True,
+        readonly=False,
+    )
+    show_followers_partner_format = fields.Char(
+        string="Partner format",
+        related="company_id.show_followers_partner_format",
+        readonly=False,
+        help="Supported parameters:\n"
+        "%(partner_name)s = Partner Name\n"
+        "%(partner_email)s = Partner Email\n"
+        "%(partner_email_domain)s = Partner Email Domain\n"
+        "Format on plain HTML: &lt;(<), &gt;(>)",
```

Comment:
Same as above, just pointing it out here.

BTW, you don't really need to repeat any of the parameters that already are found in the related field, such as `translate`, `help`, `string`... These are inherited automatically. 😉

Reply:
I will remove unnecesary properties, but `help` is not being inherited.